jetty-websocket/javax-websocket-client/src/main/java/org/eclipse/jetty/websocket/javax/client/JavaxWebSocketClientContainer.java

```diff
@@ -114,6 +114,8 @@ protected WebSocketCoreClient getWebSocketCoreClient()
                 addBean(coreClient,false);
             else
                 addManaged(coreClient);
+
+            coreClient.setCustomizer(defaultCustomizer);
```

Comment:
this should also be done if we are given a non null `CoreClient` in the constructor